pkgs/development/python-modules/pandoc/default.nix

```
@@ -0,0 +1,30 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, plumbum
+, ply
+}:
+
+buildPythonPackage rec {
+  pname = "pandoc";
+  version = "2.3";
```

There is no clear statement by upstream what version are supported but I suggest to stick to supported Python releases. The chance is very small that somebody will test it on Python <= 3.7.

```suggestion
  version = "2.3";
  format = "setuptools";

  disabled = pythonOlder "3.7";
```

---

Changed. Project's dependencies suggest that it should work with Python 3.6 or higher. Does nixpkgs have some automatic crawler that will bump this value when Python 3.7 is EOL?

---

It seems python 3.7 is no longer maintained in nixpkgs. I'll drop the `disabled` attribute.